i2c/bmp280_i2c/README.adoc

```diff
@@ -0,0 +1,41 @@
+= Attaching a BMP280 temp/pressure sensor via I2C
+
+This example code shows how to interface the Raspberry Pi Pico with the popular BMP280 temperature and air pressure sensor manufactured by Bosch. A similar variant, the BME280, exists that can also measure humidity. There is another example that uses this device but talks to it via SPI as opposed to I2c.
```

Comment:
I2c - > I2C

In the last sentence, is "this device" referring to BMP280 or BME280?

Reply:
Clarified this